### ISSUE TITLE
Csvs

### ISSUE THREAD

**Issue by ajdneuro12** [PacktPublishing/Hands-On-Gradient-Boosting-with-XGBoost-and-Scikit-learn#2]
Hello,

Thanks for this great resource. When I open the bike .csv there are no values, is there a way to fix this, thanks, see attached
[bike_rentals.csv](https://github.com/PacktPublishing/Hands-On-Gradient-Boosting-with-XGBoost-and-Scikit-learn/files/8181446/bike_rentals.csv)
!

**Comment by damvantai** [comment]
please go to that file in github and download it directly

**Comment by Newtoniano** [comment]
For anyone wondering, the data files of this repo are handled by [Github LFS](https://git-lfs.github.com/).
What you need to do is:

1.  Install the package `git-lfs` using your preferred package manager (or see the [official repo](https://github.com/git-lfs/git-lfs) for instructions on how to do it for your system
2. Clone the book repository to your desired folder with `git clone`
3. Make sure you are inside the cloned folder and run [`git lfs fetch --all`](https://github.com/git-lfs/git-lfs/blob/main/docs/man/git-lfs-fetch.1.ronn)

All the files will be downloaded to your locally cloned repository. Much more practical than downloading everything manually.
@coreyjwade it would probably be useful if you added some simple instructions for this somewhere

**Comment by coreyjwade** [comment]
Not sure which chapter you are on, but for Ch. 2, and any other github CSV
file, click "view raw" then copy in the url as follows:

url = '
https://media.githubusercontent.com/media/PacktPublishing/Hands-On-Gradient-Boosting-with-XGBoost-and-Scikit-learn/master/Chapter02/bike_rentals_cleaned.csv
'
import pandas as pd
df=pd.read_csv(url)
df.head()


Corey Wade
Director & Founder
Berkeley Coding Academy <http://berkeleycodingacademy.com>

**Comment by Newtoniano** [comment]
That  works (and that's also what is recommended in the book). However, being able to clone and download all the repo (including the data) at once is also handy, and that's exactly what git-zfs allows. If people don't read the instructions in your book carefully and simply clone the repo, you will not get the data files unless you use git-zfs on top of cloning. That's most likely what OP did, and it got me confused for a bit too. 

Thank you so much for writing this awesome book btw, it's an invaluable resource for people who are just starting out with ML like me, and want to get their hands straight into it without getting bored by too much abstract theory. Personally I found it a terrific learning resource, keep it up!

**Comment by coreyjwade** [comment]
Thanks!

**Comment by Salgat** [comment]
@coreyjwade unfortunately I'm getting this error:

`Cloning into 'C:\Users\salga\My Drive\Projects\Jupyter\Hands-On Gradient Boosting with XGBoost and scikit-learn (2020)\Hands-On-Gradient-Boosting-with-XGBoost-and-Scikit-learn'...
remote: Enumerating objects: 374, done.        
remote: Counting objects: 100% (56/56), done.        
remote: Compressing objects: 100% (28/28), done.        
remote: Total 374 (delta 43), reused 28 (delta 28), pack-reused 318 (from 1)        
Receiving objects: 100% (374/374), 140.07 MiB | 37.75 MiB/s, done.
Resolving deltas: 100% (203/203), done.
Downloading Chapter01/bike_rentals.csv (69 KB)
Error downloading object: Chapter01/bike_rentals.csv (c37fd91): Smudge error: Error downloading Chapter01/bike_rentals.csv (c37fd915a375aabaf026c3b5efac46dd7490a817c1a56b6e9b8ca83d2537e261): batch response: This repository exceeded its LFS budget. The account responsible for the budget should increase it to restore access.`

I can still manually download the full individual files through the github UI.
